CMP1903M Assessment 1 Base Code/Program.cs

```diff
@@ -18,42 +18,93 @@ class Program
     {
         internal static void Main()
         {
+            // create analysis and input output objects
+            var input = new Input();
+            var analysis = new Analyse();
 
-            try
+
+            //Create 'Input' object
+            //Get either manually entered text, or text from a file
+            Console.WriteLine("1. Enter manually\n2. Enter from text file\n1 or 2: ");
+
+            while (true)
             {
-                //Create 'Input' object
-                //Get either manually entered text, or text from a file
-                Console.WriteLine("1. Enter manually\n2. Enter from text file\n1 or 2: ");
-                var input = Convert.ToString(Console.ReadLine());
-                if (input == "1")
+                var input_mode = Convert.ToString(Console.ReadLine());
```

Comment:
We know input_mode is going to be a `string`, so we should make its type `string` as opposed to `var`.
Also implement use of `??` operator.
```suggestion
                string input_mode = Console.ReadLine() ?? string.Empty;
```